automated/linux/deadline_test/deadline_test.yaml

```diff
@@ -0,0 +1,45 @@
+metadata:
+    name: deadline_test
+    format: "Lava-Test Test Definition 1.0"
+    description: "This program is used to test the deadline scheduler
+    (SCHED_DEADLINE tasks)"
+    maintainer:
+        - anders.roxell@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+        - openembedded
+    scope:
+        - performance
+        - preempt-rt
+    devices:
+        - all
+
+params:
+    # Base interval of thread in us.
+    INTERVAL: "1000"
+    # Step size in us.
+    STEP: "500"
+    # Number of threads to test
+    # A value of "0" will run a thread per available CPU.
+    THREADS: "1"
+    # Execute cyclicdeadline for given time
+    DURATION: "5m"
+    # Background workload to be run during the meassurement
+    BACKGROUND_CMD: ""
+    # Specify url and token for publishing artifacts.
+    # For safety reasons, please set 'ARTIFACTORIAL_TOKEN' variable in job definition with
+    # 'secrets' dictionary, and set job visibility to personal or group.
+    # Refer to https://validation.linaro.org/static/docs/v2/publishing-artifacts.html
+    ARTIFACTORIAL_URL: "https://archive.validation.linaro.org/artifacts/team/qa/"
+    ARTIFACTORIAL_TOKEN: ""
+    ITERATIONS: 1
+
+run:
+    steps:
+        - cd ./automated/linux/cyclicdeadline/
+        - ./cyclicdeadline.sh -i "${INTERVAL}" -s "${STEP}" -t "${THREADS}" -D "${DURATION}" -I "${ITERATIONS}" -w "${BACKGROUND_CMD}"
```

Comment:
why doesn't this match deadline_test.sh?

Reply:
I'll drop this patch for now, needs more testing.